pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
```

Comment:
The gitignore_parser package triggers a warning that is exposed in purest. This ignores the warning. I have found no functional effect of the warning.